Testing/data/mesh_warp/mesh_warp3b.particles

```diff
@@ -0,0 +1,128 @@
+10.2514    8.05156    2.53488
+4.40765    1.90326    9.17335
+-14.5399    -1.41277    -6.49326
+19.5991    -0.650251    -0.727077
+-19.3113    -1.95056    -0.155146
+9.57796    -8.11827    2.95727
+-19.6559    -0.258554    -0.624261
+10.4303    2.09861    -8.11418
+-15.3483    6.03509    1.26782
+14.5836    6.45261    1.46638
+-15.381    0.336883    6.14303
+19.6612    -0.0492546    0.234505
+-0.855266    -9.87538    0.382723
+1.99514    -9.38314    -2.93836
+4.07457    9.23857    -2.8558
+3.54756    -2.21173    -9.48246
+1.37949    2.33253    6.1757
+4.03825    -1.7152    5.34277
+-14.7787    -6.06144    2.39872
+6.7104    6.36429    6.77721
+-1.17206    -4.16336    8.9513
+19.6093    -0.130342    -0.875756
+-5.93042    0.456613    -9.42421
+6.86613    -5.536    -7.43701
+-6.38086    9.32416    0.783857
+19.5737    0.904529    0.412452
+-7.08102    2.78779    8.77839
+14.6856    3.32431    5.67269
+-5.64882    -6.99711    -6.37619
+9.16117    -8.67471    -1.13746
+-2.61399    9.38232    2.84048
+-0.643431    3.65054    -9.17948
+-0.458466    3.27009    9.31704
+2.43217    3.70328    0.400284
+-9.40482    7.81926    3.75414
+13.3455    4.5452    -5.6843
+-13.8476    -6.93329    -1.14457
+19.6547    0.333516    -0.418724
+-10.7554    -0.536275    -8.27824
+8.20428    8.81834    -1.73737
+-11.2538    2.39315    7.74598
+13.8634    0.450619    -7.00008
+-15.5352    5.28936    -2.96435
+19.5807    0.435808    0.921458
+4.79253    -7.87042    5.46661
+-4.47631    -9.30714    -2.43978
+-0.379574    7.91447    -5.90411
+-4.01907    -3.28158    -9.11342
+2.8668    0.188998    -1.55408
+2.92608    -5.09034    3.89823
+-19.6126    -0.859056    0.0739957
+11.3656    5.47848    5.93505
+-10.9118    -5.12054    6.44739
+13.9759    -3.61624    5.95401
+-3.77394    5.5327    -7.9844
+7.09456    -0.270449    -9.21298
+-15.8616    3.96419    4.30667
+3.27679    8.06457    5.46462
+-3.34789    5.14741    8.28265
+13.7168    0.327632    7.10369
+-2.78519    -8.99136    3.8792
+19.6565    0.369836    0.119015
+1.55236    9.84213    0.412358
+7.13004    5.06294    -7.69477
+-14.7782    -3.2087    5.69722
+2.87268    5.14811    4.19701
+-10.9736    4.16212    -7.08014
+19.578    -0.384228    0.843869
+-19.6512    -0.0573275    0.525887
```

Comment:
This particle seems a little suspicious... 🤨

Reply:
I don't know what you mean, but these particles are garbage anyway, they just happened to produce the crash.